streaming/base/storage/download.py

```diff
@@ -314,6 +314,25 @@ def download_from_azure_datalake(remote: str, local: str) -> None:
         raise
 
 
+def download_from_dbfs(remote: str, local: str) -> None:
+    """Download a file from remote Databricks File System to local.
+
+    Args:
+        remote (str): Remote path (dbfs).
+        local (str): Local path (local filesystem).
+    """
+    from databricks.sdk import WorkspaceClient
```

Comment:
Can we wrap the import statement as below since `databricks` is an extra dependencies ? You can also add a method `get_import_exception_msg(pkg_name)` (or better name) in `streaming.base.util.py` file and call below as `e.msg = get_import_exception_msg(e.name)` !

```python
  try:
      from databricks.sdk import WorkspaceClient
  except ImportError as e:
      e.msg = f'Streaming was installed without {e.name} support. To use {e.name} related ' + \
              f'packages, with Streaming, run `pip install \'mosaicml-streaming[{e.name}]\'`.'
      raise e
```

Reply:
Done